### DISCUSSION TITLE
New Terminal logger for .NET 8

### DISCUSSION THREAD

**Discussion by giscus** [linkdotnet/Blog.Discussions#61] [General]
# New Terminal logger for .NET 8

With preview 4 of .NET 8, a new terminal logger was introduced. It basically removes a lot of the noise that the default logger produces. It also adds some nice colors to the output. Let's see how to use it!

https://steven-giesel.com/blogPost/f6e0accf-9493-4cc4-8992-e8300e6b0cbc

**Comment by ghadzhigeorgiev** [comment]
Superb!
